spec/till_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../app/lib/till'
+
+describe 'a till' do
+
+  let(:till) { Till.new }
+
+  describe 'taking an order' do
+
+    it 'should produce a list of items available on the menu' do
+      expect(till.menu_read_items.kind_of?(Array)).to be true
```

Comment:
I wouldn't specifically check for an array, maybe just check here to make sure that the menu is not empty. The reason is we might change `Array` to another object but it should still response to `emtpy?` - have a google about duck typing